charts/sonarqube-dce/templates/ingress.yaml

```diff
@@ -1,13 +1,7 @@
 {{- if .Values.ingress.enabled -}}
 {{- $serviceName := include "sonarqube.fullname" . -}}
 {{- $servicePort := .Values.service.externalPort -}}
-{{- if .Capabilities.APIVersions.Has "networking.k8s.io/v1" }}
 apiVersion: networking.k8s.io/v1
-{{ else if .Capabilities.APIVersions.Has "networking.k8s.io/v1beta1" }}
-apiVersion: networking.k8s.io/v1beta1
```

Comment:
We are kinda dropping the support for two apiVersion. Shall we document it in the changelog?

Reply:
Resolved see ongoing discussion [here](https://github.com/SonarSource/helm-chart-sonarqube/pull/270#discussion_r1009176505)